test/test_browser.py

```diff
@@ -1895,8 +1895,12 @@ def test_gles2_emulation(self, args):
   def test_clientside_vertex_arrays_es3(self):
     self.btest('clientside_vertex_arrays_es3.c', reference='gl_triangle.png', args=['-sFULL_ES3', '-sUSE_GLFW=3', '-lglfw', '-lGLESv2'])
 
-  def test_emscripten_api(self):
-    self.btest_exit('emscripten_api_browser.c', args=['-sEXPORTED_FUNCTIONS=_main,_third', '-lSDL'])
+  @parameterized({
+    'base': (['-sEXPORTED_FUNCTIONS=_main,_third', '-lSDL'],),
```

Comment:
You can use used the empty string here for the base base.

Reply:
Done